### PR TITLE
Show dependent submodule fails to import into Mixed Framework.

### PR DESCRIPTION
**Pull request by asa**
Building with ``` bazel build -c opt //examples/ios/Mixed:MixedFramework``` and importing the resulting Mixed.framework into a vanilla ios project results in a ```Missing required module 'submodule/``` error.

### CODE REVIEW

**Comment by thii** [comment]
This is currently impossible due to the generated module map can't be customized. Though there's a workaround that you can try: adding `@_exported import submodule` to a Swift file in the `Mixed` target.

**Comment by asa** [comment]
I couldn't get that to work.

**Comment by asa** [comment]
I missunderstood the static-ness of the mixed_static_framework to mean that its deps would be entirely contained within the resulting framework.  I am trying to split up the project into many submodules and yet deliver one single framework to thirdparty developers to include into their projects.  Is this possible with this rule or should I be looking elsewhere?

**Comment by thii** [comment]
I think it's possible. Let me fix your example.

File: examples/ios/Mixed/SwiftGreeter.swift [hunk 1]
```diff
@@ -13,6 +13,7 @@
 // under the License.
 
 import Foundation
+import submodule
```
Comment (thii) [inline examples/ios/Mixed/SwiftGreeter.swift]: Can you try again with this change? It worked for me.

```suggestion
@_implementationOnly import submodule
```

Comment (asa) [reply examples/ios/Mixed/SwiftGreeter.swift]: Yes!! That works.   Where can I find more information about these cool @_ swift tricks?

What do you want me to do with this pr?

Comment (thii) [reply examples/ios/Mixed/SwiftGreeter.swift]: Probably the Swift forums: https://forums.swift.org/t/update-on-implementation-only-imports/26996.

Could you make this a separate example (e.g. MixedWithPrivateSubmodule)? Then I'll merge it.

Comment (asa) [reply examples/ios/Mixed/SwiftGreeter.swift]: done

**Review by thii — APPROVED** [review]
Thank you!

**Pull request merged**